webapp-django/crashstats/supersearch/models.py

```diff
@@ -216,14 +232,15 @@ def get(self, **kwargs):
 
 class SuperSearchFields(ESSocorroMiddleware):
 
-    implementation = super_search_fields.SuperSearchFields
+    # Read it in once as a class attribute since it'll never change
+    # unless the .json file on disk changes and if that happens you
+    # will have reloaded the Python process.
+    _fields = SuperSearchFieldsWithoutConfig().get()
 
     API_WHITELIST = None
 
-    # The only reason this data will change is if a user changes it via the UI.
-    # If that happens, the cache will be reset automatically. We can thus
-    # increase the cache a lot here.
-    cache_seconds = 60 * 60 * 24  # 24 hours
+    def get(self):
+        return self._fields
```

Comment:
How expensive would it be to return a deep copy of that fields object here instead?